test/test.js

```diff
@@ -57,11 +57,11 @@ describe('Address encoder', () => {
 
   });
 
-  describe('[NEGATIVE] Should throw error properly', () => {
+  // describe('[NEGATIVE] Should throw error properly', () => {
```

Comment:
Why commented these out?

Reply:
@blackevil245 Lint fails because `results`, `error` and `requiredFields` fields are not used anywhere
```
/Users/laardee/maas/address-encoder/test/test.js
  62:9   error  'results' is defined but never used         no-unused-vars
  63:9   error  'error' is defined but never used           no-unused-vars
  65:11  error  'requiredFields' is defined but never used  no-unused-vars

✖ 3 problems (3 errors, 0 warnings)
```
What kind of a test do you have planned there?

Reply:
Argg i thought i have written negatives tests for these.
Basically i'll try to cover cases where address failed validation.